src/main/java/com/mucommander/ui/autocomplete/TextFieldCompletion.java

```diff
@@ -77,6 +77,11 @@ public TextFieldCompletion(AutocompleterTextComponent comp, Completer completer)
 
     	autocompletedtextComp.getDocument().addDocumentListener(documentListener);
 
+        // In previous versions of muCommander, the 'home' and 'end' keys were forwarded
+        // to the pop-up list (if opened), which I found surprising. This version handles
+        // the 'home' and 'end' keys always in the text field, like Firefox and others do.
+        boolean SHOULD_HOME_AND_END_KEYS_WORK_ON_POPUP_LIST = false;
```

Comment:
This is not needed. if we don't allow the user to change that value (and I think we don't need to allow that), then let's not make it configurable because it makes the code below more cumbersome.